src/workers/postUpdated.ts

```diff
@@ -182,22 +197,42 @@ const updatePost = async ({
 
   if (
     !databasePost ||
-    databasePost.metadataChangedAt.toISOString() >= updatedDate.toISOString()
+    databasePost.metadataChangedAt.toISOString() >=
+      data.metadataChangedAt.toISOString()
   ) {
     return null;
   }
 
   const title = data?.title || databasePost.title;
-  const updateBecameVisible = !databasePost.visible && !!title?.length;
+  const updateBecameVisible =
```

Comment:
Don't think this is particularly right, would mean we always set freeform to true, which might be the case now. but in some instances it might actually be set to false for whatever reason.
I think we can keep title.length, but take the DB value for freeform ones?

Reply:
Changed it to use the database value.